From 5e5f01604c9c2ac3e14b012563a0e9fe766ac3a7 Mon Sep 17 00:00:00 2001
From: Vladimir Barinov <vladimir.barinov@cogentembedded.com>
Date: Tue, 30 May 2017 17:41:21 +0300
Subject: [PATCH 812/909] drm: adv7511: Enable HPD interrupts to support
 hotplug

commit 1bb61bdd98bc8980abd6359d5a839af3f4d75a2f from
https://github.com/CogentEmbedded/meta-rcar.git

This patch enables HPD (hot plug detect) interrupt support

Signed-off-by: Andrey Gusakov <andrey.gusakov@cogentembedded.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/adv7511/adv7511_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c b/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
index dbc464259a5f..69f605b99553 100644
--- a/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
+++ b/drivers/gpu/drm/bridge/adv7511/adv7511_drv.c
@@ -27,7 +27,7 @@ static const struct reg_sequence adv7511_fixed_registers[] = {
 	{ 0x98, 0x03 },
 	{ 0x9a, 0xe0 },
 	{ 0x9c, 0x30 },
-	{ 0x9d, 0x61 },
+	{ 0x9d, 0x01 },
 	{ 0xa2, 0xa4 },
 	{ 0xa3, 0xa4 },
 	{ 0xe0, 0xd0 },
-- 
2.17.1

